### ISSUE TITLE
Twitch severely broken with Purple enabled

### ISSUE THREAD

**Issue by Nareese** [arthurbolsoni/Purple-adblock#94]
Sometime in the last week or so, Twitch has updated their blocker detection so that having this extension enabled COMPLETELY breaks the site.  Symptoms include:

1/ Cannot login!  Attempts to login with this extension enabled return an "unsupported browser" error, and the login fails.
2/ Chat badges are not displayed
3/ Cannot send to chat

Disabling the extension removes all issues.  If you enable the extension after logging in, you stay logged in, but the chat issues return. 

Using latest Chrome with the chrome extension.

Edit to add:  ~~The userscript version does not appear to have this issue.~~ Spoke too soon, the userscript version appears to be the same.

**Comment by hikingmetalpunk** [comment]
Seeing this also, in firefox. 

I'm still logged in but twitch won't send chats, and much of channel info won't load. Likewise going to Following and other panels just gets a "error loading data" message

**Comment by Ballasi** [comment]
Same symptoms here with Linux/Firefox.

**Comment by arthurbolsoni** [comment]
I'm looking for a solution.

**Comment by arthurbolsoni** [comment]
currently logging out and logging in is working, so I believe there must be some checking on the front-end.

**Comment by Nareese** [comment]
> currently logging out and logging in is working, so I believe there must be some checking on the front-end.

I just checked this again on my end, and the symptom remains the same, logging out and back in results in: 
![image](https://github.com/arthurbolsoni/Purple-adblock/assets/10347279/a38e4fe8-f56b-45d6-9bfc-e2d46dc8e001)

and the login fails.

**Comment by thesefer** [comment]
```Uncaught (in promise) ApolloError: Maximum call stack size exceeded

Uncaught (in promise) Error: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received```

Was wondering what's going on, totally forgot about PBA.

**Comment by TsengFayt** [comment]
Seeing the same results as others. Tried clearing cache and cookies, logging out and in, updated to the newest Chrome version and uninstalled and reinstalled the extension with the latest version, still breaks page loading. 

The stream itself loads, but the stream info and panels fail to load and the chat becomes read-only. Disabling the extension is the only fix I have found so far.

**Comment by arthurbolsoni** [comment]
> Seeing the same results as others. Tried clearing cache and cookies, logging out and in, updated to the newest Chrome version and uninstalled and reinstalled the extension with the latest version, still breaks page loading.
> 
> The stream itself loads, but the stream info and panels fail to load and the chat becomes read-only. Disabling the extension is the only fix I have found so far.

I released version 2.6.5 with the fix.
make sure you are on version 2.6.5.

**Comment by Nareese** [comment]
The Chrome extension just got updated on the chrome store this morning, but I've been using the 2.6.5 userscript version for the last couple of days and can confirm that it seems to resolve the issues for me.
